jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -188,6 +188,73 @@ bundle exec rspec $(cat /tmp/tests-to-run)
+
+Using this example, here is a quick example of how you can accomplish test splitting:
+```
+- run:
+    command: |
+      # get test files while ignoring __init__ files
+      TESTFILES=$(circleci tests glob "catalog/tests/*.py" | sed 's/\S\+__init__.py//g')
+      echo $TESTFILES | tr ' ' '\n' | sort | uniq > circleci_test_files.txt
+      cat circleci_test_files.txt
+      TESTFILES=$(circleci tests split --split-by=timings circleci_test_files.txt)
+      # massage filepaths into format manage.py test accepts
+      TESTFILES=$(echo $TESTFILES | tr "/" "." | sed 's/.py//g')
+      echo $TESTFILES
+      pipenv run python manage.py test --verbosity=2 $TESTFILES  
+```
+
+## Using test splitting with pytest
+{: #using-test-splitting-with-pytest }
+
+If you try to split your tests across containers with pytest, you may encounter any of the following errors:
+
+```
+No timing found for "tests/commands/__init__.py"
+No timing found for "tests/commands/test_1.py"
+No timing found for "tests/commands/test_2.py"
+```
+
+If any of these errors are returned, you may need to make a few adjustments, which are listed below.
+
+### Are you setting a custom working_directory?
+{: #are-you-setting-a-custom-working-directory? }
+
+If so, you may need to adjust the file paths that are saving to your test metadata XML file. Alternatively, if you are able to, try working out of the standard working directory we set for a container to see if that helps (you can do this by removing any instances of `working_directory` in your test run job).
+
+### Where does your `pytest.ini` live?
+{: #where-does-your-pytest-ini-live? }
+
+To ensure test splitting performs correctly, make sure you are running your tests in the root directory. If your tests are not being run in the root directory, you may need to run the following command before you test the `run` command:
+
+```
+cp -f .circleci/resources/pytest_build_config.ini pytest.ini
+```
+
+The `.circleci/resources/pytest_build_config.ini` path may need to be replaced to point to where it's located in your project.
+
+### Are you setting the junit_family in your pytest.ini?
+{: #are-you-setting-the-junit-family-in-your-pytest-ini? }
+
+Check to see if you have something like `junit_family=legacy` set in your pytest.ini file. For more information on how to set junit_family, refeer to the following page, which can be found [here](https://docs.pytest.org/en/stable/_modules/_pytest/junitxml.html)
+
+Search for "families" to see the relevant information. 
+
+### Example project that correctly splits by timings
+{: #example-project-that-correctly-splits-by-timing }
+
+The following GitHub repository is a fork of our `sample-python-cfd` project(https://github.com/CircleCI-Public/sample-python-cfd) that demonstrates how you can implement test splitting:
+
+https://github.com/nbialostosky/sample-python-cfd/blob/split-test/.circleci/config.yml
```

Comment:
Maybe we could make this a markdown link, by changing it to:

"This [GitHub repository is a fork](https://github.com/nbialostosky/sample-python-cfd/blob/split-test/.circleci/config.yml) ... " ?

Reply:
Actually, on second thought, maybe we should just copy+paste the config into the document, since we don't know when the original repo will be updated, and relying on a third party fork might not be idea? I checked it out and it's about 30 lines, so that might not be too bad to just include in the doc.

Reply:
Good idea....I will go ahead and add the config. Thanks!